app/components/controls/slider/DateSlider.tsx

```diff
@@ -0,0 +1,74 @@
+import { Container, Slider, styled, Typography } from "@mui/material";
+import dayjs from "dayjs";
+import { useMap } from "react-leaflet";
+
+interface DateSliderProps {
+  value: number | number[];
+  onChange: (e: Event, value: number | number[]) => void;
+  min: number;
+  max: number;
+}
+
+// Renders a basic slider (one handle) if value prop holds a single number value
```

Comment:
Should this be a doc comment? See https://tsdoc.org/.

Reply:
Sure, I'll change it to be a doc comment!